datasets/au/dfat_sanctions/crawler.py

```diff
@@ -43,6 +43,7 @@ def clean_date(date):
         "Between",
         "circa",
         "Circa",
+        "_x000D_",
```

Comment:
I don't even want to know what happened there...
<img width="1010" height="622" alt="image" src="https://github.com/user-attachments/assets/a13349b4-c6b6-4238-a859-678f56888d4a" />